boxsdk/object/item.py

```diff
@@ -176,6 +176,10 @@ def get_shared_link(self, access=None, etag=None, unshared_at=None, allow_downlo
             and has permission to set expiration dates.
         :type unshared_at:
             :class:`datetime.date` or None
+        :param password:
+            The password required to view this link. If no password is specified, the default password will be used.
```

Comment:
Is this correct? I thought that not specifying a password creates the shared link with no password.

Also, can you mention here that setting passwords is a premium feature, as is https://github.com/box/box-python-sdk/blob/master/boxsdk/object/folder.py#L192